src/ScottPlot/Plottable/RadialGaugePlot.cs

```diff
@@ -0,0 +1,709 @@
+            set => _GaugeSpacePercentage = (value > 100 ? 100 : (value < 0 ? 0 : value));
+        }
+        private float _GaugeSpacePercentage = 50f;
+
+        /// <summary>
+        /// <see langword="Color"/> of the value labels drawn inside the gauges.
+        /// </summary>
+        public Color GaugeLabelsColor { get; set; } = Color.White;
+
+        /// <summary>
+        /// Size of the gague label text as a percentage of the gauge width.
+        /// Values in the range [0-100], default value is 75 [percent]. Other values might produce unexpected side-effects.
+        /// </summary>
+        public float GaugeLabelsFontPct
+        {
+            get => _GaugeLabelsFontPct;
+            set => _GaugeLabelsFontPct = (value > 100 ? 100 : (value < 0 ? 0 : value));
+        }
+        private float _GaugeLabelsFontPct = 75f;
+
+        /// <summary>
+        /// <see langword="Font"/> used for labeling values on the plot
+        /// </summary>
+        public Drawing.Font Font { get; set; } = new();
+
+        /// <summary>
+        /// <see langword="True"/> if value labels are shown inside the gauges.
+        /// Size of the text is set by <see cref="GaugeLabelsFontPct"/> and color by <see cref="GaugeLabelsColor"/>.
+        /// </summary>
+        public bool ShowGaugeValues { get; set; } = true;
+
+        public System.Drawing.Drawing2D.LineCap EndCap { get; set; } = System.Drawing.Drawing2D.LineCap.Triangle;
+
+        public System.Drawing.Drawing2D.LineCap StartCap { get; set; } = System.Drawing.Drawing2D.LineCap.Round;
+
+        /// <summary>
+        /// Controls rendering style of the concentric circles (ticks) of the web
+        /// </summary>
+        //public RadarAxis AxisType { get; set; } = RadarAxis.None;
+
+        // These 3 properties are needed as part of IPlottable
+        public bool IsVisible { get; set; } = true;
+        public int XAxisIndex { get; set; } = 0;
+        public int YAxisIndex { get; set; } = 0;
+
+        #endregion Properties & fields
+
+        /// <summary>
+        /// Initializes the plot instance.
+        /// </summary>
+        /// <param name="values">Array of values to be plotted as gauges.</param>
+        /// <param name="lineColors">Array colors for the gauges.</param>
+        public RadialGaugePlot(double[] values, Color[] lineColors)
+        {
+            GaugeColors = lineColors;
+            Update(values);
+        }
+
+        public override string ToString() =>
+            $"RadialGauge with {DataRaw.Length} points.";
+
+        /// <summary>
+        /// Replace the data values with new ones. This passed data is copied and stored in <see cref="DataRaw"/>.
+        /// Implicitly calls the <see cref="ComputeAngularData"/> routine
+        /// </summary>
+        /// <param name="values">Array of values to be plotted as gauges.</param>
+        public void Update(double[] values)
+        {
+            DataRaw = new double[values.Length];
```

Comment:
How come we need to copy this instead of just assigning it? Do we mutate it?

Reply:
No, I believe that data is not mutated throughout the routines.
I was afraid of just keeping a reference to the data values. Since `DataRaw` is used several times in different functions, I thought there was a change the user could modify (or dispose of) the original data values; thus causing exceptions whenever that reference would be used. Copying data would avoid that.

Reply:
> data is not mutated throughout the routines

I'll probably refactor data handling toward a functional style where `ComputeMaxMin()` and `ComputeAngularData()` are `static` and return tuples. 

This one-way type of data processing (rather than manipulating global state) is a little easier to extend and troubleshoot.